instructure/canvas-ios
# Add syllabus downloader

refs: MBL-16767
affects: Student
release note: none

test plan:
- Sync the syllabus tab for offline usage.
- Turn off internet on the device.
- Go to syllabus page.
- Syllabus html content should display along with any assignments if syllabus summary is enabled.

## Checklist
